src/store/CrosshairStore.js

```diff
@@ -380,16 +376,16 @@ class CrosshairStore {
     };
 
     onContextReady = () => {
-        this.crosshair = this.stx.layout.crosshair;
+        this.state = this.stx.layout.crosshair;
```

Comment:
IIRC mutating the `@observable` state could immediately trigger mobx to fire a change event. Better to store it in a temporary variable until it is confirmed to change:

```js
const state = this.stx.layout.crosshair;
this.state = (typeof state !== 'number') ? 0 : state;
```